org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/ClasspathUpdateHandler.java

```diff
@@ -22,29 +22,27 @@
 import org.eclipse.jdt.core.IJavaElementDelta;
 import org.eclipse.jdt.core.IJavaProject;
 import org.eclipse.jdt.core.JavaCore;
-import org.eclipse.jdt.ls.core.internal.ActionableNotification;
+import org.eclipse.jdt.ls.core.internal.EventNotification;
+import org.eclipse.jdt.ls.core.internal.EventType;
 import org.eclipse.jdt.ls.core.internal.JavaClientConnection;
 import org.eclipse.jdt.ls.core.internal.ProjectUtils;
-import org.eclipse.lsp4j.MessageType;
 
 public class ClasspathUpdateHandler implements IElementChangedListener {
 
-	public static final String CLASSPATH_UPDATED_NOTIFICATION = "__CLASSPATH_UPDATED__";
```

Comment:
I wonder if someone else will be broken by that

Reply:
Hmm, Not sure about that.

At least for the VS Code side, the client itself handles the logic of receiving the notification, the exposed event is still the same.